pkgs/applications/video/davinci-resolve/default.nix

```diff
@@ -165,8 +167,8 @@ let
           name = "davinci-resolve";
```

Comment:
Shouldn't this be `name = pname;` (in case someone installs both versions as the icon comment alludes to)?

Reply:
You're right; also edited `desktopName` to distinguish the two.